src/test/java/com/mycompany/coffesort/SortingConsecutivesTest.java

```diff
@@ -0,0 +1,115 @@
+package com.mycompany.coffesort;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import org.junit.Assert;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+
+/**
+ * Test class for the SortingConsecutives class.
+ *
+ * @author marylene
+ * @since 9 January 2020
+ */
+public class SortingConsecutivesTest {
+
+    @Test
+    /**
+     * This will test that the given array is sorted correctly using insertion
+     * sort by shifting the elements
+     *
+     * @covers coffeesort.SortingConsecutives.insertionSortShifting()
+     */
+    public void testMinimumSwaps() {
+        System.out.println("minimumSwaps");
+
+        List<int[][]> testScenarios = new ArrayList(); //a list of test scenarios
+
+        testScenarios.add(new int[][]{{}, {0}});// empty
+        testScenarios.add(new int[][]{{1}, {0}});// one element
+        testScenarios.add(new int[][]{{1, 2}, {0}});// two elements; sorted
+        testScenarios.add(new int[][]{{3, 1, 2}, {2}});// three elements
+        // simple test
+        testScenarios.add(new int[][]{{2, 3, 4, 1, 5}, {3}});
+
+        // a simple test
+        testScenarios.add(new int[][]{{4, 3, 1, 2}, {3}});
+        // another simple test
+        testScenarios.add(new int[][]{{7, 1, 3, 2, 4, 5, 6}, {5}});
+
+        // Random tests
+        for(int i = 0; i < 10; i++) {
+            
+            
+            int[] randArray = shuffle();
+            // Create 'deepCopy' of array
+            int[] testArray = Arrays.copyOf(randArray, randArray.length);
+            testScenarios.add(new int[][]{randArray,{minimumSwaps(testArray)}});
```

Comment:
These tests are okay but seem like an overkill. Probably better to create a sort method that returns a sorted array in a separate class and test it.